packages/pipeline-editor/src/PipelineController/index.ts

```diff
@@ -455,11 +466,11 @@ class PipelineController extends CanvasController {
 
     if (node !== undefined) {
       const { op, app_data } = node;
-      const nodeDef = this.nodes.find((n) => n.op === op);
+      const nodeDef = this.getAllPaletteNodes().find((n) => n.op === op);
 
       const info = nodeDef?.properties?.uihints?.parameter_info ?? [];
 
-      const properties = info.map((i) => {
+      const properties = info.map((i: any) => {
```

Comment:
once we define a real type in `getAllPaletteNodes` we should be able to remove `any` here